File: src/main/kotlin/net/wavem/uvc/rms/domain/job/result/JobResult.kt
package net.wavem.uvc.rms.domain.job.result

data class JobResult(
    val status: String,
    val startTime: String,
    val endTime: String,
    val startBatteryLevel: Int,
    val endBatteryLevel: Int,
    val dist: Int
) {
}

File: src/rms/response/event/domain/taskInfo/jobResult/job_result.py

class JobResult():
    
    def __init__(
        self,
        status: str = None, 
        startTime: str = None, 
        endTime: str = None, 
        startBatteryLevel: int = None,
        endBatteryLevel: int = None,
        dist: int = None
    ) -> None:
        self.status = status
        self.startTime = startTime
        self.endTime = endTime
        self.startBatteryLevel = startBatteryLevel
        self.endBatteryLevel = endBatteryLevel
        self.dist = dist
        

__all__ = ['job_result']